microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
 
-## Roadmap
+## Evergreen distribution mode  
 
-We recognize that the Edge browser may not be available on all user machines where your application will run, or that installing Edge browser throughout your organization may be difficult. To ensure your application runs on all machines, independent of the installed Microsoft Edge browser, we will release the Microsoft Edge WebView2 Runtime. Additionally, we will update WebView2 to search for the stable version of the runtime before searching for pre-release versions of the installed browser.
+The evergreen distribution mode ensures that your app is taking advantage of the latest features and security updates.  The evergreen distribution mode has the following characteristics.  
 
-There will be support for two distribution options using the WebView2 Runtime: evergreen, and fixed version.
+*   The web platform updates automatically without additional effort from you.  
+*   All applications that leverage the evergreen distribution mode use a shared copy of the platform binaries, which saves disk space.  
 
-Evergreen will be the recommended distribution model for most developers. In this mode, updates are pushed automatically to the WebView2 Runtime without additional effort from your application. The evergreen model ensures that your app is taking advantage of the latest features and security updates for hosted web content.
+There are multiple channels WebView2 that applications may use as the web platform.  By default, WebView2 targets the most stable channel available on the device that meets the minimum version requirement of the WebView2 SDK.  The following channels are listed in order from most to least stable channel.  
 
-For constrained environments there will be support for a fixed version distribution model. In this model, your application selects and packages a specific version of WebView2. Updates to the WebView version do not occur automatically, and will be the responsibility of the application. The fixed version model is beneficial when you need to control the browser version, and when your application updates. 
+1.  WebView2 Runtime \(Preview\)  
+1.  Microsoft Edge Beta Channel  
+1.  Microsoft Edge Dev Channel  
+1.  Microsoft Edge Canary Channel    
 
-### Microsoft Edge WebView2 Runtime
+> [!NOTE]
+> The evergreen distribution model is recommended for most developers.  
 
-The Microsoft Edge WebView2 Runtime will package the browser binaries optimized for use by WebView2 applications. It will function standalone or side-by-side with a client Edge Browser installed. Installing the runtime once will support many WebView2 applications running on the client machine. Installing the runtime will not appear as a browser to end-users, and will have no desktop shortcuts, start menu entry point, or protocol registration.
+Microsoft Edge Stable Channel runtime is not a valid target for WebView2, and the reasons are described later.  
```

Comment:
Mark as IMPORTANT. Also, delete the word runtime after `Microsoft Edge Stable Channel`

Reply:
Updated.
```md
> [!IMPORTANT]
> Microsoft Edge Stable Channel is not a valid target for WebView2, and the reasons are described later.  
```